e2e/test/check-volume-detached.sh

```diff
@@ -72,20 +32,21 @@ for ((i=1; i<=$MAX_RETRIES; i++)); do
     if [ $? -eq 0 ]; then
         # Check if the response is valid JSON
         if jq -e . >/dev/null 2>&1 <<< "$response"; then
-            # Extract linode_id and check if it's null
+            # Extract linode_id & volume_name
             linode_id=$(echo "$response" | jq -r '.data[0].linode_id')
+            volume_name=$(echo "$response" | jq -r '.data[0].label' | sed 's/^pvc//')
             
```

Comment:
Since we are relying on the linode_id and volume_id in response to GET call and not passing the namespace and pv names, the tests will fail if that account has any instance with a volume attached. I was running tests on my account where I already have one instance which has a PV attached to it. The tests keep failing as they always find one instance with a PV attached.
```
=== NAME  chainsaw/pod-pvc-create-ext4-filesystem
    | 23:08:17 | pod-pvc-create-ext4-filesystem | Check if the volume is detached on Node resource and in Linode (using API) | SCRIPT    | LOG   |
        === STDOUT
        Checking Linode API for volume status...
        Volume still attached in Linode. Response:
        {"data": [{"id": 4816548, "status": "active", "label": "pvc186e668a8dd24e06", "created": "2024-07-01T21:21:22", "updated": "2024-07-01T21:21:51", "filesystem_path": "/dev/disk/by-id/scsi-0Linode_Volume_pvc186e668a8dd24e06", "size": 30, "linode_id": 60903693, "linode_label": "lke196119-281877-01eb6f1e0000", "region": "ap-southeast", "tags": [], "hardware_type": "nvme"}, {"id": 4836650, "status": "active", "label": "pvcfe2ce4e3b57248a5", "created": "2024-07-03T03:06:44", "updated": "2024-07-03T03:06:59", "filesystem_path": "/dev/disk/by-id/scsi-0Linode_Volume_pvcfe2ce4e3b57248a5", "size": 10, "linode_id": 60950110, "linode_label": "csi-driver-cluster-6a210d0-control-plane-vznwl", "region": "us-ord", "tags": ["chainsaw-enabling-wildcat"], "hardware_type": "nvme"}, {"id": 4836652, "status": "active", "label": "pvcd611bf8c23af41fd", "created": "2024-07-03T03:06:46", "updated": "2024-07-03T03:07:32", "filesystem_path": "/dev/disk/by-id/scsi-0Linode_Volume_pvcd611bf8c23af41fd", "size": 10, "linode_id": null, "linode_label": null, "region": "us-ord", "tags": ["chainsaw-peaceful-glowworm"], "hardware_type": "nvme"}], "page": 1, "pages": 1, "results": 3}
        Retrying in 5 seconds...
 ...
        === STDERR
        Max retries reached. Volume is still attached in Linode.
    | 23:08:17 | pod-pvc-create-ext4-filesystem | Check if the volume is detached on Node resource and in Linode (using API) | SCRIPT    | ERROR |
        === ERROR
        ($error): Invalid value: "exit status 1": Expected value: <nil>
        (contains($stdout, 'Volume detached in Linode')): Invalid value: false: Expected value: true
        (contains($stdout, 'Volume was successfully detached')): Invalid value: false: Expected value: true
...
chainsaw-enabling-wildcat
--- FAIL: chainsaw (0.00s)
    --- FAIL: chainsaw/pod-pvc-basic-filesystem (103.57s)
    --- FAIL: chainsaw/pod-pvc-expand-storage (105.32s)
    --- FAIL: chainsaw/pod-pvc-expand-raw-block-storage (115.38s)
    --- FAIL: chainsaw/statefulset-pvc (81.95s)
    --- FAIL: chainsaw/pod-pvc-create-ext4-filesystem (99.19s)
FAIL
Tests Summary...
- Passed  tests 0
- Failed  tests 5
- Skipped tests 0
Done with failures.
Error: some tests failed
make: *** [Makefile:121: e2e-test] Error 1
Error: error running script "e2e-test" in Devbox: exit status 2 
```

Reply:
ah thats right. let me fix that

Reply:
Should be fixed now! I missed a change in the script which was causing that to happen. Thanks for catching that rahul!